ingestion-edge/pytest.ini

```diff
@@ -7,7 +7,6 @@ filterwarnings =
 # https://github.com/googleapis/google-cloud-python/blob/92465cb/pubsub/google/cloud/pubsub_v1/publisher/client.py#L76
     ignore::PendingDeprecationWarning:google.cloud.pubsub_v1.publisher.client:89
     ignore::PendingDeprecationWarning:google.cloud.pubsub_v1.subscriber.client:83
-docstyle_match = (?!test_)(?!conftest).*\.py$
```

Comment:
pytest-docstyle became pytest-pydocstyle, which requires pydocstyle to be configured directly